bench/MainCriterion.hs

```diff
@@ -44,8 +44,9 @@ main =
   defaultMainWith
     (defaultConfig {resamples = 100})
     [ env
-      -- fawltyContent <- T.pack <$> readFile "data/ttl/fawlty1.ttl"
         (do rdfContent <- T.pack <$> readFile "bills.099.actions.rdf"
+            fawltyContentTurtle <- T.pack <$> readFile "data/ttl/fawlty1.ttl"
+            fawltyContentNTriples <- T.pack <$> readFile "data/nt/all-fawlty-towers.nt"
             let (Right rdf1) =
                   parseString (XmlParser Nothing Nothing) rdfContent
             let (Right rdf2) =
```

Comment:
Is there any reason to not just replace all uses of rdf2 with rdf1? they are guaranteed to be identical.